### PR TITLE
MODCLUSTER-493 Add support for Tomcat 8.5; merge into one module also…

### PR DESCRIPTION
**Pull request by rhusar**
… supporting Tomcat 9

Jira
https://issues.jboss.org/browse/MODCLUSTER-493

### CODE REVIEW

**Comment by modcluster-pull-request** [comment]
Triggering build using a merge of e92bfaec804ace6fd03f3732a7b488b665054408 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/

**Comment by modcluster-pull-request** [comment]
Build 187 outcome was SUCCESS using a merge of e92bfaec804ace6fd03f3732a7b488b665054408 on branch master:
Private: https://jenkins.mw.lab.eng.bos.redhat.com/hudson/job/mod_cluster-1.3.x-pull-player-executor/187
Public: https://hudson.jboss.org/hudson/job/mod_cluster-1.3.x-pull-player-executor/187

File: container/pom.xml [hunk 1]
```diff
@@ -43,7 +43,7 @@
                 <module>tomcat6</module>
                 <module>tomcat7</module>
                 <module>tomcat8</module>
-                <module>tomcat9</module>
+                <module>tomcat85</module>
```
Comment (pferraro) [inline container/pom.xml]: Why not call this tomcat8.5?

Comment (jfclere) [inline container/pom.xml]: The repo in ASF is tomcat85...

Comment (pferraro) [inline container/pom.xml]: Fair enough.

**Pull request merged**